wiki/CreatingMissions.md

```diff
@@ -346,6 +346,10 @@ If this mission is to be shown at `landing`, it shows up as soon as you land ins
 
 A mission shown when `assisting` or `boarding` will be shown when you repair a friendly ship or plunder a hostile ship, respectively. **These missions are never shown when boarding a ship that you have boarded before, that belongs to you, or that is an NPC in an active mission.** In either case, if the `on offer` conversation results in a conversation exit code of `launch` or `flee`, the ship in question will be destroyed.
 
+An `"in flight"` mission is offered after you have taken off from a planet or wormhole, or after you have finished jumping to a system. The `source` filter can be used to filter which systems the mission can offer in, instead of filtering for planets. **(v. 0.10.13)**
```

Comment:
Can it also offer when a relevant condition changes? (And if not today, could this behavior still be added in the future?)